ddtrace/ext/aws.py

```diff
@@ -27,6 +28,33 @@ def unpacking_args(args, args_name, traced_args_list):
     return response
 
 
+def flatten_args(args):
```

Comment:
We might want this to return a `dict` instead, then we can just do `span.set_tags(aws.flatten_args(args))`